cmd/snap/cmd_auto_import.go

```diff
@@ -100,8 +134,19 @@ func autoImportFromAllMounts() (int, error) {
 
 	added := 0
 	for _, cand := range cands {
-		if err := ackFile(cand); err != nil {
+		err := ackFile(cand)
+		// HORRIBLE: string compare OMG
+		// the server is not ready yet
+		if err != nil && strings.Contains(err.Error(), "cannot communicate with server") {
+			logger.Noticef("queuing for later %s", cand)
```

Comment:
We should probably point out that this was the case in stderr. Otherwise it'll look to the user as if it had worked, and it'll be surprising to not observe its effects.

It's probably not even worth logging it. That'll be the normal working conditions of this functionality, and it'll be boring to see that every time the USB key is attached just because it has that file.